packages/lodestar/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@chainsafe/lodestar",
```

Comment:
Thoughts on renaming this to `lodestar-chain` or `lodestar-beacon` or something like that?

Reply:
reason?

Reply:
tbh - i kind of back this idea to some degree... Lodestar has sort of evolved from this massive beast, to a small army of individual components.

Reply:
^ That. Lodestar will eventually include the light client or something too. Helps distinguish what is what

Reply:
I agree, "lodestar" is sort of a nebulous term now. I think `lodestar-beacon` makes sense.
Related, we could move cli into a separate package and reexport all relevant objects with a 'convenient' interface in the `lodestar` package.